DataFormats/BTauReco/interface/PFCombinedTauTagInfo.h

```diff
@@ -36,11 +36,11 @@ namespace reco {
       signedSflightpath_=NAN;
       PFChargedHadrCandsEtJetEtRatio_=NAN;
       PFNeutrHadrCandsE_=NAN;
-      PFNeutrHadrCandsN_=numeric_limits<int>::quiet_NaN();
+      PFNeutrHadrCandsN_=std::numeric_limits<int>::quiet_NaN();
```

Comment:
[not asking for a change, just a note]
someone was particularly confusing here: 
- double PFNeutrHadrCandsE_ is assigned an implementation-defined float NAN
- int PFNeutrHadrCandsN_ is assigned a generic programming nan which has standard language value of 0